### DISCUSSION TITLE
Recuperar repo pasado

### DISCUSSION THREAD

**Discussion by Juanaa16** [sebasvelasco353/FDP-2022_01#48] [Q&A]
hola profe, te comento ya un poco mas calmada. hoy a las 2 am cuando estaba intentando subir desde la consola mi repo con la carpeta de la clase 9 actualizada (osea, con el trabajo terminado) no se que hice mal (porque hice git getch, git pull, y git push) que lo que hizo fue que me actualizó LA CARPETA LOCAL a como estaba en el repo web, osea, cuando no llevaba nada de trabajo. quiero saber como puedo recuperar mi repo pasado, intenté de todo, con revert, buscar en los commits pasados pero es que ni siquiera aparece como un commit, y tenía todo listo y ya no tengo prácticamente nada, necesito recuperar mi repo de ayer antes de tocar la consola para agregar la entrega, literal no tengo nada que entregar por eso, gracias.

**Comment by sebasvelasco353** [comment]
Hola Juana, puedes decirme porfa en que rama del repositorio estas?
eso lo puedes ver en la consola dandole git branch o desde vscode en la parte inferior izquierda... puede ser que te cambiaste de rama.

**Reply by Juanaa16** [reply]
hola profe, le di a lo de git branch en el FDP y me dice main

**Reply by Juanaa16** [reply]
tambien dice main en vscode :(

**Reply by sebasvelasco353** [reply]
puedes usar git log y mostrarme lo que te sale porfa

**Reply by Juanaa16** [reply]
Mira, me sale esto[](url
<img width="1440" alt="Screen Shot 2022-02-16 at 7 22 24 PM" src="https://user-images.githubusercontent.com/98325588/154379973-6ff4329a-677f-4be2-b0cd-77abf6ad4ace.png">
)

**Reply by sebasvelasco353** [reply]
habias hecho un commit antes de hacer el fetch upstream?

**Reply by Juanaa16** [reply]
agh, no, ya entiendo que pasó, se me pasó con el sueño que tenía
